pkg/distro/rhel9/edge.go

```diff
@@ -197,6 +201,11 @@ var (
 		"NetworkManager.service", "firewalld.service", "sshd.service", "fdo-client-linuxapp.service",
 	}
 
+	disabledServices = []string{
+		//grub-boot-success interfering with greenboot
+		"grub-boot-success.timer",
```

Comment:
@say-paul with this change in RHEL9 have you tested it correctly disables the user service?

Reply:
Yes , tested it on rhel 9.